src/lib/api/documents.test.ts

```diff
@@ -17,9 +17,14 @@ const test = base.extend({
 });
 
 describe("document fetching", () => {
-  test.todo("get a single document");
+  test.todo("documents.get");
+  test.todo("documents.search");
+});
 
-  test.todo("search documents");
+describe("document uploads and processing", () => {
+  test.todo("documents.create");
+  test.todo("documents.upload");
+  test.todo("documents.process");
```

Comment:
Since these fns have been implemented, this is the right time to write these tests against them. We can also `test.todo("documents.cancel")` since that's stubbed.